NBitcoin.Tests/transaction_tests.cs

```diff
@@ -3412,5 +3412,31 @@ private byte[] ParseHex(string data)
 		{
 			return Encoders.Hex.DecodeData(data);
 		}
+
+		[Fact]
+		public void ShouldSendAll()
+		{
+			var builder = Network
+				.CreateTransactionBuilder();
+
+			var k = new Key();
+			var dest = k.PubKey.Hash;
+
+			var rate = new FeeRate(Money.Coins(0.0004m));
+			builder
+				.AddCoins(RandomCoin(Money.Coins(0.5m), k))
+				.AddCoins(RandomCoin(Money.Coins(0.5m), k))
+				.AddKeys(k)
+				.SendAll(dest);
+			
+			var fee = builder.EstimateFees(rate);
+			builder.SendFees(fee);

```

Comment:
Why no `SendEstimatedFees`?

Reply:
Originally just because I wasn't aware of the api. Now because `fee` is required for assertion.